tests/testthat/test-type-data-frame.R

```diff
@@ -110,8 +110,9 @@ test_that("combining data frames with foreign classes uses fallback", {
     foo <- structure(mtcars[1:3], class = c("foo", "data.frame"))
     bar <- structure(mtcars[4:6], class = c("bar", "data.frame"))
     baz <- structure(mtcars[7:9], class = c("baz", "data.frame"))
-    expect_warning(vec_ptype_common_df_fallback(foo, bar, baz))
-    expect_warning(vec_ptype_common_df_fallback(foo, baz, bar, baz, foo, bar))
+    # Nested expect_warning() require testthat > 2.3.2
+    suppressWarnings(expect_warning(vec_ptype_common_df_fallback(foo, bar, baz)))
```

Comment:
How about repeated expect-warning? I assume the number of warnings is deterministic?

Reply:
This works for the dev version of testthat, not for the CRAN version.